<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FavoritesWidget_teacher</class>
 <widget class="QWidget" name="FavoritesWidget_teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>649</width>
    <height>569</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>649</width>
    <height>569</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>收藏夹</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FavoritesWidget_teacher{
background-color: rgb(235, 244, 234);
}
QLineEdit {
    border-width: 1px;
    padding: 1px;
    border-style: solid;
	border-color: rgb(22, 102, 57);
    border-radius: 5px;
}

/* As mentioned above, eliminate the padding and increase the border. */
QLineEdit:focus:focus {
    border-width: 3px;
    padding: 0px;
}


QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}
.QPushButton:!hover
{
background-color: rgb(162, 210, 152);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);
border-radius:3px;
 padding: 3px;
}
.QPushButton:hover
{
background-color: rgb(85, 170, 127);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);
border-radius:3px
}
.QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
}</string>
  </property>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/search.png);
background-color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>32</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>公告信息</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>学校文件</string>
    </attribute>
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>党务工作</string>
    </attribute>
    <widget class="QListView" name="listView_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab4">
    <attribute name="title">
     <string>教育教学</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab5">
    <attribute name="title">
     <string>学生工作</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab6">
    <attribute name="title">
     <string>后勤工作</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab7">
    <attribute name="title">
     <string>讲座论坛</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab8">
    <attribute name="title">
     <string>人才招聘</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab9">
    <attribute name="title">
     <string>科研事项公示</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="dropButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消收藏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
